README.md

```diff
@@ -1141,7 +1141,7 @@ fi
 You can export `GPG_TTY` like this anywhere in `~/.zshrc`:
 
 ```zsh
-export GPG_TTY=$TTY
+export GPG_TTY=${TTY}
```

Comment:
There are many unrelated changes in this PR, such as this line:

```diff
- export GPG_TTY=$TTY
+ export GPG_TTY=${TTY}
```

Please revert them and keep only quoting changes, only adding a double quote here and there.

Reply:
Done. I did keep one change unrelated to double quotes.

```
-git clone --depth=1 https://github.com/romkatv/powerlevel10k.git $ZSH_CUSTOM/themes/powerlevel10k
+git clone --depth=1 https://github.com/romkatv/powerlevel10k.git "${ZSH_CUSTOM:-$HOME/.oh-my-zsh/custom}/themes/powerlevel10k"
```

As I believe the fallback was missed and should be included. If you prefer it to be a separate PR, that's fine, I'm happy to open one; just let me know.